tools/BUILD.bazel

```diff
@@ -22,12 +22,9 @@ exports_files(["CROSSTOOL"])
     name = "cc-gcc-" + cpu,
     all_files = ":empty",
     compiler_files = ":empty",
-    cpu = cpu,
```

Comment:
I'm fine having these changes in this PR, but can you put them in a separate commit?